buckbuild.bzl

```diff
@@ -1463,6 +1463,7 @@ def define_buck_targets(
             "torch/csrc/jit/mobile/train/random.cpp",
             "torch/csrc/jit/mobile/train/sequential.cpp",
             ":gen_aten_libtorch[autograd/generated/Functions.cpp]",
+            "torch/csrc/quantized/quantized_backward.cpp",
```

Comment:
I'm still surprised this is not in tools/build_variables.bzl. But if you only need it here it's ok. But that means that this won't be included in other targets and so you might get strange behavior when these backwards are not registered there.

Reply:
Hmmmm where should I place the code then?

Reply:
I'm not 100% sure tbh but I would have put it with similar files in https://github.com/pytorch/pytorch/blob/5b2ab0dd4f9b75657ba0e83641cd91864b4f23d1/build_variables.bzl#L156